src/Microsoft.DotNet.Cli/Program.cs

```diff
@@ -22,9 +22,12 @@ public class Program
   -v|--verbose  Enable verbose output
 
 Common Commands:
+  init          Initialize a basic .NET project
+  restore       Restore dependencies specified in the .NET project
   compile       Compiles a .NET project
-  publish       Publishes a .NET project for deployment
-  run           Compiles and immediately executes a .NET project";
+  publish       Publishes a .NET project for deployment (including the runtime)
+  run           Compiles and immediately executes a .NET project
+  pack          Package a NuGet package of the application";
```

Comment:
Creates NuGet package or creates a package of the application (drop the word NuGet)